modules/atomic-widgets/prop-types/background-image-overlay-prop-type.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace Elementor\Modules\AtomicWidgets\PropTypes;
+
+use Elementor\Modules\AtomicWidgets\PropTypes\Base\Object_Prop_Type;
+use Elementor\Modules\AtomicWidgets\PropTypes\Primitives\String_Prop_Type;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+
+class Background_Image_Overlay_Prop_Type extends Object_Prop_Type {
+	public static function get_key(): string {
+		return 'background-image-overlay';
+	}
+
+	protected function define_shape(): array {
+		return [
+			'url' => String_Prop_Type::make(),
+			'repeat' => String_Prop_Type::make(),
```

Comment:
```suggestion
			'url' => Url_Type::make(),
			'repeat' => String_Prop_Type::make(),
```

and don't you have more restrictions for the repeat?

Maybe it could be an union:

/* Keyword values */
background-repeat: repeat;
background-repeat: repeat-x;
background-repeat: repeat-y;
background-repeat: space;
background-repeat: round;
background-repeat: no-repeat;

/* Two-value syntax: horizontal | vertical */
background-repeat: repeat space;
background-repeat: repeat repeat;
background-repeat: round space;
background-repeat: no-repeat round;

/* Global values */
background-repeat: inherit;
background-repeat: initial;
background-repeat: revert;
background-repeat: revert-layer;
background-repeat: unset;